bin/download_submission.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+import sys
+
+import synapseclient
+
+submission_id = sys.argv[1]
```

Comment:
Should this new script follow the same thing you've set up for the other scripts using the arg parser, ie:


```

def get_args():
    # implementation

if __name__ == "__main__":
    args = get_args()
    # implementation
```
?

Reply:
I didn't feel the need to do so since it is such a simple script, but for consistency I will make this change